src/inference/src/threading/ie_istreams_executor.cpp

```diff
@@ -334,6 +334,30 @@ Parameter IStreamsExecutor::Config::GetConfig(const std::string& key) const {
     return {};
 }
 
+void IStreamsExecutor::Config::UpdateHybridCustomThreads(Config& config) {
+    const auto num_cores = parallel_get_max_threads();
+    const auto num_cores_phys = getNumberOfCPUCores();
+    const auto num_big_cores_phys = getNumberOfCPUCores(true);
+    const auto num_big_cores = num_cores > num_cores_phys ? num_big_cores_phys * 2 : num_big_cores_phys;
+    const auto num_small_cores_phys = num_cores_phys - num_big_cores_phys;
+    const auto threads = config._threads ? config._threads : num_cores;
+    const auto streams = config._streams > 0 ? config._streams : 1;
+
+    config._small_core_offset = num_big_cores;
+    const int threads_per_stream = std::max(1, threads / streams);
+    config._threads_per_stream_big = std::min(num_big_cores_phys, threads_per_stream);
+    config._threads_per_stream_small = std::min(num_small_cores_phys, threads_per_stream);
```

Comment:
Should cover 6Pcore or 10Pcore case as below:

    const int threads_per_stream = std::max(1, threads / streams);
    const int base_big_streams = int( (num_big_cores_phys + threads_per_stream - 1)/ threads_per_stream)  * 2;
    config._threads_per_stream_big = std::min(num_big_cores_phys, int(num_big_cores / base_big_streams));
    config._threads_per_stream_small = std::min(num_small_cores_phys, config._threads_per_stream_big);

Reply:
Ok, done.